crypto/src/ed25519.rs

```diff
@@ -354,11 +372,11 @@ impl KeyPair for Ed25519KeyPair {
         }
     }
 
-    fn generate<R: rand::CryptoRng + rand::RngCore>(rng: &mut R) -> Self {
-        let kp = ed25519_dalek::Keypair::generate(rng);
+    fn generate<R: rand::CryptoRng + rand::RngCore>(_rng: &mut R) -> Self {
+        let kp = ed25519_consensus::SigningKey::new(OsRng);
```

Comment:
i plan to upgrade rand and replace all OsRng in one PR with https://github.com/MystenLabs/narwhal/issues/544 to keep this one focused